utils/appliance/__init__.py

```diff
@@ -2172,6 +2174,48 @@ def configure_appliance_for_openldap_ext_auth(self, appliance_fqdn):
             'setenforce 0 && systemctl restart sssd && systemctl restart httpd')
         self.wait_for_web_ui()
 
+    @logger_wrap("Configuring VM Console: {}")
+    def configure_vm_console_cert(self, log_callback=None):
+        """This method generates a self signed SSL cert and installs it
+           in the miq/vmdb/certs dir.   This cert will be used by the
+           HTML 5 VM Console feature.  Note evmserverd needs to be restarted
+           after running this.
+        """
+        log_callback('Installing SSL certificate')
+        cert = conf.cfme_data['vm_console']['cert']
```

Comment:
**Optional**
Consider using a dict.get() call here in case the value isn't specified, to control the exception flow a bit better.